onedal/neighbors/neighbors.py

```diff
@@ -349,9 +348,6 @@ def _kneighbors(self, X=None, n_neighbors=None, return_distance=True, queue=None
         # If the query data is the same as the indexed data, we would like
```

Comment:
Can remove the comment as well if the code is being removed as well

Reply:
I added some separation from the call before for clarity, but comes from here: https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/neighbors/_base.py#L910